### PR TITLE
Added arguably as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/arguably/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "arguably" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arguably-{{ version }}.tar.gz
```
Comment (martinRenou) [inline recipes/arguably/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/arguably/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/treykeown/arguably
+  doc_url: https://treykeown.github.io/arguably/
+  summary: Turns functions and docstrings into command line interfaces.
+  license: MIT
```
Comment (martinRenou) [inline recipes/arguably/meta.yaml:39]: I'm not sure about this one, it seems to be using a custom LICENSE:
https://github.com/treykeown/arguably/blob/main/LICENSE.txt

Comment (thewchan) [reply recipes/arguably/meta.yaml:39]: looks like the initial part is definitely MIT but they added the weird AI stuff in the middle paragraph.... not sure what to make of it?

Comment (thewchan) [reply recipes/arguably/meta.yaml:39]: @conda-forge/core advice needed here

Comment (ocefpaf) [reply recipes/arguably/meta.yaml:39]: While they editing to the license is terrible and, IMO, you should open an issue upstream to point it to them that. Here, all you can do is to add it as a custom license with `LicenseRef-Arguably`.

Comment (thewchan) [reply recipes/arguably/meta.yaml:39]: Sigh, maybe we don't wanna deal with this

Comment (ocefpaf) [reply recipes/arguably/meta.yaml:39]: The LicenseRef-Arguably is an "okay" path.

**Pull request closed without merging**